app.organicmaps.desktop.yml

```diff
@@ -0,0 +1,39 @@
+app-id: app.organicmaps.desktop
+runtime: org.kde.Platform
+runtime-version: '5.15'
+sdk: org.kde.Sdk
+command: OMaps-wrapper
+finish-args:
+  - --share=network
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+modules:
+  - name: organicmaps
+    buildsystem: cmake-ninja
+    builddir: true
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+    post-install:
+      - 'install -Dm755 ../wrapper.sh ${FLATPAK_DEST}/bin/OMaps-wrapper'
+    cleanup:
+      - '/lib' # static libraries and build configs
+      - '/include'
+    sources:
+      - type: git
+        # TODO set to upstream when the pr is merged
+        url: https://github.com/fgaz/organicmaps
+        # TODO on next release, add tag: YYYY.MM.DD-N-android
+        commit: fbebe70c7a4a67128cd94c6d35f29c7d65e34361
```

Comment:
I don't see were the AppStream file is installed either. Not in the CMakefile.txt nor here. (the former is prefered, it should be installed regardless)